moose_code.rb

```diff
@@ -52,4 +52,3 @@ def decode_message(message)
     decode_word(word)
   end
 end
```

Comment:
- [ ] Your methods for decoding word and sentence do not work as expected. See what they return in the screenshot below.

  I would suggest you refactor to something like this:
  ```
  # method to decode an entire word in Morse code
  def decode_word(word)
    @letters = word.split
    @current_word = ''
    @letters.each do |letter|
      @current_word += decode_char(letter)
    end
    @current_word
  end

  # method to decode an entire message in Morse code
  def decode_message(message)
    @words = message.split('   ')
    current_msg = ''
    @words.each do |word|
      current_msg += decode_word(word) + ' '
    end
    current_msg.strip
  end
  ```
  ![arr0](https://user-images.githubusercontent.com/54848363/194715321-f8eb5425-36bd-426b-ba09-50e143a16632.jpg)